locale/en.po

```diff
@@ -7044,6 +6987,206 @@ msgstr "UI"
 msgid "WIKI_HEADING_UPGRADES"
 msgstr "Upgrades"
 
+msgid "WIKI_HELPANDTIPS_INTRO"
+msgstr ""
+"[i]Here you can see basic gameplay information and general tips.[/i]\n"
+"\n"
+"\n"
+"\n"
+"\n"
+"[b][u]Basic game mechanics[/u][/b]\n"
```

Comment:
It seems these should be made into headings on the wiki so that a table of contents is created and the translation text would be split into more sensible sections...